bokehjs/test/unit/models/widgets/markup.ts

```diff
@@ -0,0 +1,108 @@
+import sinon from "sinon"
+import {expect} from "assertions"
+
+import {Div} from "@bokehjs/models/widgets/div"
+import {MathJaxProvider} from "@bokehjs/models/text/providers"
+import {find_tex} from "@bokehjs/models/text/mathjax"
+import {build_view} from "@bokehjs/core/build_views"
+import {MarkupView} from "@bokehjs/models/widgets/markup"
+
+function tex2svg(text: string, options: {display: boolean}) {
+  console.log(text, options)
+  return {outerHTML: `<svg display="${options.display}">${text}</svg>`}
+}
+
+class InternalProvider extends MathJaxProvider {
```

Comment:
Don't replicate this. For now import from axes and later move it to a common utility module.

Reply:
can I import things from integration tests inside unit test? Did you see that I'm mocking the tex2svg function of this?